src/eyepop/endpoint.ts

```diff
@@ -143,6 +144,163 @@ export class Endpoint {
         }
     }
 
+    public async manifest(): Promise<any> {
```

Comment:
You can define a type for the manifest entries in `types.ts` and preserve the illusion of type safety. I found it helpful when staying in the typescript world. This is a benefit for us internally (not so much for our customers). In go this is:

```
// swagger:model
type SourcesEntry struct {
	Authority string `json:"authority"`
	Manifest  string `json:"manifest"`
}
```